spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerReflectionAutoConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package org.springframework.grpc.autoconfigure.server;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.grpc.BindableService;
+import io.grpc.protobuf.services.ProtoReflectionService;
+
+/**
+ * {@link EnableAutoConfiguration Auto-configuration} for gRPC Reflection service
+ * <p>
+ * grpc.reflection.enabled=true must be present in configuration in order for the
```

Comment:
Let's prefix this property w/ `spring.` and also please add an entry to the [additional metadata file](https://github.com/spring-projects-experimental/spring-grpc/blob/9cf02273548b9642087ae0d5ee0d4503ae7f054d/spring-grpc-spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json#L5) that way auto-complete in editor will work for this property.

<sub>Background [here](https://docs.spring.io/spring-boot/specification/configuration-metadata/annotation-processor.html#appendix.configuration-metadata.annotation-processor.adding-additional-metadata)</sub>

Reply:
Done.